docs/pages/design_tokens.js

```diff
@@ -26,7 +26,7 @@ card(
               <Text weight="bold">CSS Token Name</Text>
             </Table.HeaderCell>
             <Table.HeaderCell>
-              <Text weight="bold">JaveScript Prop Name</Text>
+              <Text weight="bold">Javescript Prop Name</Text>
```

Comment:
Javascript typo

Reply:
hah, fixed one typo and missed the other - good catch!